packages/vapor-demo/components/components/BlankSlate.js

```diff
@@ -0,0 +1,29 @@
+const BlankSlate = () => (
+  <>
+    <div className="blankslate m2">
+      <h1>This is a blank slate</h1>
```

Comment:
Some files have 2 space indentation, some 4... Are all those files all getting parsed by prettier ?

Reply:
Prettier is not yet run in the vapor-demo project, i'll do a single PR for this at the end